markdown/ch1.markdown

```diff
@@ -719,6 +719,45 @@ def binned_boxplot(x, y,
     plt.show()
 ```
 
+
+
+
+
+TODO: make this a tip box. Code currently doesn't run using the ">" notedown format.
+Also, can't have code with a deliberate error in it or it crashes the whole thing.
+
+**Tip Box: Python 3-exclusive: using '*' to create Keyword-Only Arguments**
+
+Since version 3.0 Python has enabled us to create
+["keyword-only" arguments](https://www.python.org/dev/peps/pep-3102/).
+These are arguments that you have to call using a keyword, you can't rely on position alone.
+For example, in the `binned_boxplot` function we just wrote, you could call it like this:
+```
+binned_boxplot(x, y, xlabel='my x label', ylabel='my y label')
+```
+but not like this:
+```
+binned_boxplot(x, y, 'my x label', 'my y label')
```

Comment:
... which is possible in Python 2.  (some readers may never have seen this kind of usage before!)